README.md

```diff
@@ -2,39 +2,32 @@
 _ultimate tool for upload files to Wikimedia Commons and other Wikimedia projects_
 
 ## Usage
-For manual, take a look at [project wiki](https://github.com/yarl/vicuna/wiki) and [website](http://yarl.github.io/vicuna).
+For manual, take a look at [project wiki](https://github.com/yarl/vicuna/wiki) and [website](https://yarl.github.io/vicuna).
 
 ## Build and Run
-Program is being written using [NetBeans IDE](https://netbeans.org/) and [Apache Ant](https://ant.apache.org/) is used for building. In order to download and build source code, do following:
+Program is being written using any IDE (NetBeans preffered) you want and [Gradle](https://gradle.org/) is used for building. In order to download and build source code, do following:

```

Comment:
there's a typo, should be "preferred"

Reply:
Fixed in https://github.com/yarl/vicuna/pull/128